MapleServer2/PacketHandlers/Game/UserChatHandler.cs

```diff
@@ -5,22 +5,36 @@
 using MapleServer2.Tools;
 using Microsoft.Extensions.Logging;
 
-namespace MapleServer2.PacketHandlers.Game {
-    public class UserChatHandler : GamePacketHandler {
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class UserChatHandler : GamePacketHandler
+    {
         public override RecvOp OpCode => RecvOp.USER_CHAT;
 
         public UserChatHandler(ILogger<GamePacketHandler> logger) : base(logger) { }
 
-        public override void Handle(GameSession session, PacketReader packet) {
-            var type = (ChatType) packet.ReadInt();
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            var type = (ChatType)packet.ReadInt();
             string message = packet.ReadUnicodeString();
             string recipient = packet.ReadUnicodeString();
             packet.ReadLong();
 
             GameCommandActions.Process(session, message);
-
-            if (type == ChatType.All) {
-                session.FieldManager.SendChat(session.Player, message);
+            switch (type)
+            {
+                case (ChatType.Super):
+                    session.FieldManager.SendChat(session.Player, message, type);
+                    break;
+                case (ChatType.World):
+                    session.FieldManager.SendChat(session.Player, message, type);
+                    break;
+                case (ChatType.Channel):
+                    session.FieldManager.SendChat(session.Player, message, type);
+                    break;
+                default:
+                    session.FieldManager.SendChat(session.Player, message);
+                    break;
```

Comment:
Since you're using a switch here, you might as well explicitly pass in the type, otherwise you might as well just make all of this a single line of code if there's going to be nothing different between them